cargo-shuttle/tests/integration/init.rs

```diff
@@ -19,17 +39,68 @@ async fn cargo_shuttle_init(path: &str) -> anyhow::Result<CommandOutcome> {
                 working_directory,
                 name: None,
             },
-            cmd: Command::Init(InitArgs { path }),
+            cmd: Command::Init(InitArgs {
+                axum: false,
+                rocket: true,
+                tide: false,
+                tower: false,
+                path,
+            }),
         })
         .await
 }
 
-#[tokio::test]
-async fn basic_init() {
-    cargo_shuttle_init("tests/tmp/basic-init").await.unwrap();
+#[cfg(test)]
+mod init_integration_tests {
```

Comment:
The `cfg` check and namespacing should not be needed since these are in the `tests` directory